### PR TITLE
Increase Rewards Tests Coverage

### PR DESCRIPTION
**Pull request by narayanprusty**
## Description

Resolves #VEN-825

## Checklist

- [x] I have updated the documentation to account for the changes in the code.
- [x] If I added new functionality, I added tests covering it.
- [x] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [x] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

**Review by defcon022 — APPROVED** [review]

**Review by chechu — COMMENTED** [review]
Would it make sense to check the specific values we would expect? Not only checking they are bigger than 0. These values shouldn't change with changes in the implementation. They are defined by business rules. This way we would check that we implemented these business rules properly

File: tests/hardhat/Rewards.ts [hunk 1]
```diff
@@ -39,11 +40,10 @@ let fakePriceOracle: FakeContract<PriceOracle>;
 let fakeAccessControlManager: FakeContract<AccessControlManager>;
 
 describe("Rewards: Tests", async function () {
-  const RewardsDistributor = await ethers.getContractFactory("RewardsDistributor");
   /**
    * Deploying required contracts along with the poolRegistry.
    */
-  before(async function () {
+  beforeEach(async function () {
```
Comment (kkirka) [inline tests/hardhat/Rewards.ts:46]: Might be quite slow. Maybe move it into a fixture?

**Comment by narayanprusty** [comment]
> Would it make sense to check the specific values we would expect? Not only checking they are bigger than 0. These values shouldn't change with changes in the implementation. They are defined by business rules. This way we would check that we implemented these business rules properly

I agree. I have calculated the rewards and updated the tests.

File: tests/hardhat/Rewards.ts (hunk 1 above)
Comment (narayanprusty) [reply tests/hardhat/Rewards.ts]: Done

Comment (narayanprusty) [reply tests/hardhat/Rewards.ts]: Done

**Review by chechu — APPROVED** [review]

**Pull request merged**